optimum/exporters/tasks.py

```diff
@@ -1400,9 +1424,8 @@ def infer_task_from_model(
             subfolder (`str`, *optional*, defaults to `""`):
                 In case the model files are located inside a subfolder of the model directory / repo on the Hugging
                 Face Hub, you can specify the subfolder name here.
-            revision (`Optional[str]`, *optional*):
+            revision (`Optional[str]`, *optional*, defaults to `None`):
```

Comment:
Feel free to remove the *optional* here, cf https://huggingface.slack.com/archives/C02P0559X9S/p1669628754896019

Reply:
removed